lib/engine-addon.js

```diff
@@ -664,10 +665,24 @@ module.exports = {
 
         var engineJSTree = buildEngineJSTreeWithoutRoutes.call(this);
 
+        // If includeRoutesInApplication is false, separate the routes.js file
+        // and its dependencies
+        if (this.lazyLoading.includeRoutesInApplication === false) {
```

Comment:
Can we create a variable `var separateRoutes = this.lazyLoading.includeRoutesInApplication === false;`?

I think it would make the code easier to read and allows us to only perform this check in one location.